src/EPR.RegulatorService.Frontend.Web/ViewModels/RegistrationSubmissions/GrantRegistrationSubmissionViewModel.cs

```diff
@@ -1,10 +1,12 @@
-namespace EPR.RegulatorService.Frontend.Web.ViewModels.RegistrationSubmissions
+namespace EPR.RegulatorService.Frontend.Web.ViewModels.RegistrationSubmissions;
+
+using System.ComponentModel.DataAnnotations;
+
+public class GrantRegistrationSubmissionViewModel
 {
-    using EPR.RegulatorService.Frontend.Web.Attributes;
-    using EPR.RegulatorService.Frontend.Web.Constants;
-
-    public class GrantRegistrationSubmissionViewModel
-    {
-        public Guid OrganisationId { get; set; } 
-    }
-}
+    public Guid OrganisationId { get; set; }
+
+    [Required(ErrorMessage = "ErrorMessage")]
```

Comment:
Should this be a locale key name?  Shared perhaps?  What is output to the user?

Reply:
This is a locale key name in the GrantRegistrationSubmission.en.resx. Below is the ouput of how it looks:
![image](https://github.com/user-attachments/assets/bcecb72f-3e26-46d0-ab03-d41cea3febd0)